mesonbuild/backend/backends.py

```diff
@@ -506,6 +506,8 @@ def write_test_serialisation(self, tests, datafile):
             is_cross = self.environment.is_cross_build() and \
                 self.environment.cross_info.need_cross_compiler() and \
                 self.environment.cross_info.need_exe_wrapper()
+            if isinstance(exe, (build.CustomTarget, build.BuildTarget)):
+              is_cross = is_cross and exe.is_cross
```

Comment:
__[flake8]__
* [E111] indentation is not a multiple of four

<!-- Comment by [SideCI](https://sideci.com) -->